contributors/devel/sig-testing/e2e-tests.md

```diff
@@ -575,7 +581,9 @@ to be eligible for this tag. This tag does not supersed any other labels.
   - `[LinuxOnly]`: If a test is known to be using Linux-specific features
 (e.g.: seLinuxOptions) or is unable to run on Windows nodes, it is labeled
 `[LinuxOnly]`. When using Windows nodes, this tag should be added to the
-`skip` argument.
+`skip` argument. This is not using `[Feature:LinuxOnly]` because that
+would have implied changing all CI jobs which skip tests with unknown
+requirements.
```

Comment:
`[LinuxOnly]` is kind of a dumpster fire ([at least in the networking tests](https://github.com/kubernetes/kubernetes/issues/124426)) and maybe should be revisited/refactored anyway...

Reply:
Agreed, but I'll leave that to others.

Reply:
And probably it is better to do it in smaller steps, i.e. just clarify `Feature` vs. `FeatureGate` now and start using Ginkgo labels.